pkgs/development/python-modules/pyside6/test.nix

```diff
@@ -0,0 +1,88 @@
+# FIXME "pip install" cannot find PySide6
+# it works when i remove install_requires
+
+{ lib
+, python3
+, qt6
+, fetchFromGitHub
+}:
+
+{
+  python-pyside6-hello = python3.pkgs.buildPythonApplication rec {
+    pname = "qt6-pyside-test";
+    version = "0.0.1";
+
+    src = fetchFromGitHub {
+      owner = "GRomR1";
+      repo = "python-pyside6-hello";
+      rev = "1531b409168ff530300a7fed0c33387c635ff235";
+      sha256 = "Y12z8ex/WU/Bc55RidvylPh8wytL8mXMjSMHd433Yos=";
+    };
+
+    postUnpack = ''
+      (
+      cd $sourceRoot
+
+      # add shebang line
+      (
+        echo '#! /usr/bin/env python3'
+        cat main.py
+      ) >${pname}
+      rm main.py
+
+      # add setup.py
+      cat >setup.py <<'EOF'
+      from setuptools import setup
+      setup(
+        name='${pname}',
+        version='${version}',
+        scripts=['${pname}'],
+
+        # FIXME
+        #install_requires=['PySide6'],
+        # ERROR: Could not find a version that satisfies the requirement PySide6
```

Comment:
fixme

Reply:
@milahu i fixed this in the latest push by doing the same thing arch does in pyside6 and shiboken6 https://github.com/archlinux/svntogit-packages/blob/e1fe2063534324b8d978bb3a5e0b687f6e07f8ea/trunk/PKGBUILD#L34

Reply:
thanks!

weird they still use *.egg format ... but ok